addenda/txp.go

```diff
@@ -196,6 +196,38 @@ func ParseTXP(paymentInfo string) (*TXP, error) {
 	return txp, nil
 }
 
+// ToString serializes the TXP object into a TXP-formatted string
+// The format matches the expected TXP addenda format:
+// TXP*tax_id*tax_type*date*type1*amount1*type2*amount2*type3*amount3*taxpayer_verification\
+func (txp *TXP) ToString() string {
```

Comment:
The name `String() string` would match this library better. We don't want String() and ToString() - that'd be confusing.